.github/ISSUE_TEMPLATE/user_story.md

```diff
@@ -21,8 +18,27 @@ assignees: ''
 
 <!-- What is the point of this user story ? Why is it necessary ? -->
 
+## Definition of ready
+
+<!-- All these items are checked by an implementation team so it can begin work -->
+<!-- DO NOT CHECK THESE YOURSELF -->
+
+- [ ] The user story is well defined
+- [ ] Acceptance criterias are defined
+- [ ] The individual responsible for validating the user story is assigned
+- [ ] The user story can be shown off
```

Comment:
what about : 
"The team should understand how to provide a demo of the features"

Reply:
"The team should understand how to demonstrate the features"